packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -740,11 +740,6 @@ export abstract class FluidDataStoreContext
 		this.makeLocallyVisibleFn();
 	}
 
-	/** @deprecated - To be replaced by calling makeLocallyVisible directly  */
-	public bindToContext() {
```

Comment:
Seems like we could have removed this sooner?  Who was calling this before? (and same for IFluidDataStoreContext.bindToContext)

Reply:
Looks like we could have, yeah